Kiosk/App/Views/RegisterFlowView.swift

```diff
@@ -21,9 +23,17 @@ class RegisterFlowView: ORStackView {
         self.updateConstraints()
     }
     
-    var titles = ["Mobile", "Email", "Credit Card", "Postal/Zip"]
```

Comment:
It might be a little less complicated to do something like `var titles = ["Mobile", "Email", "Credit Card"] + (appSetup.needsZipCode ? ["Postal/Zip"] : [])`. I think it has the advantage of working immutably, but also keeping the logic local.

I wish languages had a pattern for making items contingent in array literals, like if you could do `var titles = ["Mobile", "Email", "Credit Card", "Postal/Zip" if appSetup.needsZipCode]` or something. Scala doesn't have that either :/